### PR TITLE
<Date Picker /> - fix rtl arrows position in responsive designs

### PR DESCRIPTION
**Pull request by poseneror**
### 🔦 Summary
In rtl the navigation arrows position is wrong.
In LTR:
Screen Shot 2021-04-07 at 9 58 28 AM

In RTL:
Screen Shot 2021-04-07 at 9 58 23 AM

...

### ✅ Checklist

- 🔬 Change is tested
  - [x] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_7009/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_7009

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx [hunk 1]
```diff
@@ -154,5 +154,15 @@ visualize('DatePicker', () => {
         today={new Date('2019/12/17')}
       />,
     );
+    snap(
+      'responsive rtl',
+      <div dir="rtl">
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx:159]: <TPAComponentsConsumer>
```suggestion
      <div dir="rtl">
           <TPAComponentsProvider value={{ rtl: true }}>
```

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx [hunk 2]
```diff
@@ -154,5 +154,15 @@ visualize('DatePicker', () => {
...
+        <VisualTestDatePicker
+          width="600px"
+          value={new Date('2019/12/15')}
+          onChange={() => {}}
+        />
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx:164]:
```suggestion
        />
  </TPAComponentsProvider value={{ rtl: true }}>
```

**Review by sivanharel — CHANGES_REQUESTED** [review]
Great!
We just need to add the  TPAComponentsProvider as well here see the comment.

File: packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx [hunk 3]
```diff
@@ -154,5 +154,17 @@ visualize('DatePicker', () => {
...
+    snap(
+      'responsive rtl',
+      <div dir="rtl">
+           <TPAComponentsProvider value={{ rtl: true }}>
+        <VisualTestDatePicker
+          width="600px"
+          value={new Date('2019/12/15')}
+          onChange={() => {}}
+        />
+  </TPAComponentsProvider value={{ rtl: true }}>
```
Comment (sivanharel) [inline packages/wix-ui-tpa/src/components/DatePicker/DatePicker.visual.tsx:166]:
```suggestion
  </TPAComponentsProvider>
```

**Review by sivanharel — APPROVED** [review]

**Comment by poseneror** [comment]
Thanks @sivanharel :)

**Pull request merged**